packages/devtools_app/lib/src/profiler/cpu_profile_controller.dart

```diff
@@ -61,10 +77,13 @@ class CpuProfilerController
   ValueListenable<String> get userTagFilter => _userTagFilter;
   final _userTagFilter = ValueNotifier<String>(userTagNone);
 
-  @visibleForTesting
-  final dataByTag = <String, CpuProfileData>{};
+  Iterable<String> get userTags =>
+      cpuProfileStore.lookupProfile(label: userTagNone)?.userTags ?? [];
 
-  Iterable<String> get userTags => dataByTag[userTagNone]?.userTags ?? [];
+  bool get isToggleFilterActive =>
+      toggleFilters.firstWhere((filter) => filter.enabled.value,
```

Comment:
https://api.dart.dev/stable/2.14.1/dart-core/Iterable/any.html
does this more cleanly.

Reply:
nice. thanks.